Sources/XcodeGenKit/SourceGenerator.swift

```diff
@@ -452,12 +452,10 @@ class SourceGenerator {
                 groupChildren.append(variantGroup)
                 baseLocalisationVariantGroups.append(variantGroup)
 
-                let sourceFile = SourceFile(
-                    path: filePath,
-                    fileReference: variantGroup,
-                    buildFile: PBXBuildFile(file: variantGroup),
-                    buildPhase: .resources
```

Comment:
Previous implementation, all files under `*.lproj` are treated as `Resources`. However, they should be generated by `generateSourceFile` function.

Reply:
Should this be it's own PR?